mobile/src/main/java/com/example/android/uamp/MediaNotificationManager.java

```diff
@@ -47,6 +50,10 @@
 public class MediaNotificationManager extends BroadcastReceiver {
     private static final String TAG = LogHelper.makeLogTag(MediaNotificationManager.class);
 
+    private static final String CHANNEL_ID = "MusicApp_Channel";
```

Comment:
Please change this to be in the form of "com.example.android.uamp.channel.MUSIC_CHANNEL_ID".